test/unit/mem/view/src/ViewSubViewTest.cpp

```diff
@@ -159,6 +161,46 @@ static auto createVecFromIndexedFn()
 }
 
 //-----------------------------------------------------------------------------
+//! Prints all elements of the buffer.
+//-----------------------------------------------------------------------------
+struct PrintBufferKernel {
+    template <typename TAcc,
+              typename TIter>
+    ALPAKA_FN_ACC void operator()(
+        TAcc const & acc,
+        TIter begin,
+        TIter end) const {
+
+        (void)acc;
+        for(; begin != end; begin++){
+            std::cout << *begin << std::endl;
+        }
+    }
+};
+
+//-----------------------------------------------------------------------------
+//! Compares iterator element-wise
+//-----------------------------------------------------------------------------
+struct CompareBufferKernel {
+    template <typename TAcc,
+              typename TIterA,
+              typename TIterB>
+    ALPAKA_FN_ACC void operator()(
+        TAcc const & acc,
+        TIterA beginA,
+        TIterA endA,
+        TIterB beginB) const {
```

Comment:
Sorry for these style issues, but it is not my style of programming. Thus, it sometimes happens that I fall back into old habits.